fenix/app/src/main/res/values/strings.xml

```diff
@@ -2363,4 +2363,20 @@
     <string name="download_languages_item_content_description_not_downloaded_state">Download</string>
     <!-- Content description (not visible, for screen readers etc.): For a language list item that is selected. -->
     <string name="download_languages_item_content_description_selected_state">Selected</string>
+
+    <!-- Debug drawer -->
+    <!-- The title of the Tab manager feature in the Debug Drawer. -->
+    <string name="debug_drawer_tab_manager_title" tools:ignore="UnusedResources">Tab manager</string>
+    <!-- The section header for the tab count section in the Tab manager. -->
+    <string name="debug_drawer_tab_manager_tab_count_title">Tab count</string>
+    <!-- The normal tab count category in the tab count section in the tab manager. -->
+    <string name="debug_drawer_tab_manager_tab_count_normal">Normal</string>
+    <!-- The inactive tab count category in the tab count section in the tab manager. -->
+    <string name="debug_drawer_tab_manager_tab_count_inactive">Inactive</string>
+    <!-- The private tab count category in the tab count section in the tab manager. -->
+    <string name="debug_drawer_tab_manager_tab_count_private">Private</string>
+    <!-- The total tab count category in the tab count section in the tab manager. -->
+    <string name="debug_drawer_tab_manager_tab_count_total">Total</string>
+    <!-- A footnote to indicate inactive tabs when are disabled. -->
+    <string name="debug_drawer_tab_manager_inactive_tabs_disabled">* Inactive tabs are disabled</string>
```

Comment:
Does the `*` notation make sense if translate these?

Reply:
[It looks like we have at least one official string with this](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/strings.xml#1609).

Reply:
My comment above might mean this is not required

Reply:
collapsing this thread in favor of https://github.com/mozilla-mobile/firefox-android/pull/4859#discussion_r1431578629